code/modules/paperwork/clipboard.dm

```diff
@@ -1,154 +1,157 @@
+#define PAPERWORK	1
+#define PHOTO		2
+#define IS_PEN(x)	(istype(x, /obj/item/weapon/pen))
```

Comment:
This should go in the https://github.com/ParadiseSS13/Paradise/blob/master/code/__DEFINES/is_helpers.dm file and follow the formatting guidelines there.